src/providers/fileVariableReferencesCodeLensProvider.ts

```diff
@@ -17,9 +17,11 @@ export class FileVariableReferencesCodeLensProvider implements CodeLensProvider
         const requestRanges: [number, number][] = Selector.getRequestRanges(lines, { ignoreFileVariableDefinitionLine: false });
 
         for (let [blockStart, blockEnd] of requestRanges) {
-            while (blockStart <= blockEnd) {
+            for (; blockStart <= blockEnd; blockStart++) {
                 const line = lines[blockStart];
-                if (!Selector.isFileVariableDefinitionLine(line)) {
+                if(Selector.isCommentLine(line)) {
```

Comment:
Add space between `if` and `(`

Reply:
Done in 27caf445f49ee0039c53821c934325e97658f242